setup.py

```diff
@@ -54,6 +54,7 @@
     "coverage == 6.1.2",  # https://github.com/nedbat/coveragepy/issues/1300
     "pytest >= 6.2.2, < 7.0",  # pytest 7 incompatible with pytest-postgres < 4
     "pytest-cov >= 2.11.1",
+    "flake8 < 5.0.0",  # flake8 5 incompatible with pytest-flake8 > 1.0.7
```

Comment:
This maybe should be (was actually) in a different PR

Reply:
this has been addressed